frontend/src/components/DriverDetail/DriverDetail.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Vehicle } from '../../types';
+import styles from './driverDetail.module.css';
+import UserDetail, { UserContactInfo, OtherInfo } from '../UserDetail/UserDetail';
+import { phone, clock, wheel } from "./icons";
+
+type DriverDetailProps = {
+  // profilePic: string;
+  firstName: string;
+  lastName: string;
+  netId: string;
+  phone: string;
+  availability: string[][]
+  vehicle: Vehicle
+}
+
+const DriverDetail = (props: any) => {
+  console.log("driver detail running");
+  const driver: DriverDetailProps = props.location.state;
+  console.log("driver", driver);
+  const reducer = (acc: string, [day, timeRange]: string[]) => acc + day + ": " + timeRange + " • ";
```

Comment:
Instead of calling this reducer, consider calling it something that indicates what its function is